composer.json

```diff
@@ -27,7 +27,7 @@
     "require-dev": {
         "laminas/laminas-coding-standard": "~1.0.0",
         "phpbench/phpbench": "^0.17.1",
-        "phpunit/phpunit": "^9.3.7"
+        "phpunit/phpunit": "~9.3.7"
```

Comment:
Tests would not even run on PHPUnit 9.4, due to an error in either PHPUnit or our configuration. I've set it to the 9.3 series to ensure they run at this time.